server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -76,6 +76,11 @@ export class TenantManager {
         if (!tenant) {
             winston.error("Tenant is disabled or does not exist.");
             return Promise.reject(new Error("Tenant is disabled or does not exist."));
+        } else {
+            let externalStorageData = tenant.customData.externalStorageData;
+            if (tenant.customData.externalStorageData !== undefined) {
```

Comment:
Can we simply check if externalStorageData is truthy instead of specifically checking if it is not undefined so that we check for null cases too? in the rest of the code as well.